src/app/AppConfig.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * Centralized configuration for the application.
+ * Fetches configuration values from environment variables.
+ */
+class AppConfig {
+  /**
+   * The Graphql API URL (e.g., 'https://venepaikat-federation-gateway.api.hel.fi/').
+   *
+   * @throws {Error} If the `REACT_APP_API_URL` environment variable is not defined.
+   */
+  static get apiUrl() {
+    return getEnvOrError(process.env.REACT_APP_API_URL, 'REACT_APP_API_URL');
+  }
+  /**
+   * The OIDC (OpenID Connect) authority URL.
+   *
+   * @throws {Error} If the `REACT_APP_OIDC_AUTHORITY` environment variable is not defined.
+   */
+  static get oidcAuthority() {
+    const origin = getEnvOrError(process.env.REACT_APP_OIDC_AUTHORITY, 'REACT_APP_OIDC_AUTHORITY');
+    return new URL(origin).href;
+  }
+  /**
+   * The audiences for OIDC tokens.
+   * Can be a string or a comma-separated list of strings.
+   *
+   * @example
+   * - Tunnistamo: undefined (leave the env var empty)
+   * - Keycloak: 'berth-api-test,profile-api-test'
```

Comment:
In the `.env.example` there's `berths-api-dev,profile-api-test`, here it's berth, not berths. Is the prefix `berth-` or `berths-`?